griptape-aws-bill-pdf-to-csv/README.md

```diff
@@ -1,24 +1,18 @@
-This sample takes in a S3 URI pointing at a AWS Bill PDF and converts the bill into a CSV. Each item in the bill is converted to a row in the CSV. It will then write the CSV back to a provided S3 URI. This showcases how Griptape can be used to automate previously manual processes such as identifying if a PDF field with identical formatting is a `AWS Region` or an `AWS Service`.
+This sample takes a Bucket and Asset that represent an AWS Bill PDF stored in Griptape Cloud and converts the bill into a CSV. Each item in the bill is converted to a row in the CSV. The CSV is uploaded back to the Bucket provided as a new Asset in addition to being returned by the Structure as a ListArtifact. This sample demonstrates both Griptape Cloud's Bucket and Asset resources and how Griptape can be used to automate previously manual/hardcoded processes. One such process is identifying if a PDF value with identical formatting is a `AWS Region` or an `AWS Service`.
```

Comment:
Do we have docs on the buckets we can link out to? If not, those should get written for the generic cloud docs.

Reply:
We do not, this should be a task as part of UX V1 to document how the data lake portion of the console works.

Reply:
Those docs are primarily console oriented, I want to write a whole `Data Lake` section but need the UX for that to be accurate